nautobot/core/settings.py

```diff
@@ -1092,3 +1092,22 @@ def silk_request_logging_intercept_logic(request):
 
 
 SILKY_INTERCEPT_FUNC = silk_request_logging_intercept_logic
+
+#
+# Kubernetes settings variables
+#
+
+# File path to the config file used to load and configure the kubernetes cluster
+KUBERNETES_CONFIG_FILE = os.getenv("NAUTOBOT_KUBERNETES_CONFIG_FILE", "./development/kind-kube-config")
+
+# Name of the kubernetes pod created in the kubernetes cluster
+KUBERNETES_JOB_POD_NAME = os.getenv("NAUTOBOT_KUBERNETES_JOB_POD_NAME", "nautobot-job-pod")
+
+# Namespace of the kubernetes pod created in the kubernetes cluster
+KUBERNETES_JOB_POD_NAMESPACE = os.getenv("NAUTOBOT_KUBERNETES_JOB_POD_NAMESPACE", "default")
+
+# Nautobot container name in the kubernetes pod
+KUBERNETES_JOB_CONTAINER_NAME = os.getenv("NAUTOBOT_KUBERNETES_JOB_CONTAINER_NAME", "nautobot")
+
+# Nautobot Image name used to build a container in the kubernetes pod
+KUBERNETES_JOB_IMAGE_NAME = os.getenv("NAUTOBOT_KUBERNETES_JOB_IMAGE_NAME", "networktocode/nautobot:latest")
```

Comment:
Not sure if any/all of these should have global defaults other than `""` or `None`. Can set them to development-specific values in `development/nautobot_config.py` or `nautobot/core/tests/nautobot_config.py` as appropriate.